expat/lib/xmlparse.c

```diff
@@ -603,6 +603,9 @@ struct XML_ParserStruct {
   OPEN_INTERNAL_ENTITY *m_openInternalEntities;
   OPEN_INTERNAL_ENTITY *m_freeInternalEntities;
   XML_Bool m_defaultExpandInternalEntities;
+  int m_entityRecursionLevel;
+  long m_entityExpansionLen;
```

Comment:
That `long` should probably be `XML_Size` instead, especially for people with `XML_LARGE_SIZE` defined.